src/main/java/ftb/utils/net/MessageClaimChunk.java

```diff
@@ -1,19 +1,19 @@
 package ftb.utils.net;
 
-import latmod.lib.ByteCount;
 import cpw.mods.fml.common.network.simpleimpl.*;
 import ftb.lib.LMAccessToken;
 import ftb.lib.api.net.LMNetworkWrapper;
 import ftb.utils.world.*;
 import ftb.utils.world.claims.ClaimedChunk;
+import latmod.lib.ByteCount;
 
 public class MessageClaimChunk extends MessageFTBU {
 
     public static final int ID_CLAIM = 0;
     public static final int ID_UNCLAIM = 1;
     public static final int ID_UNCLAIM_ALL = 2;
     public static final int ID_UNCLAIM_ALL_DIMS = 3;
-    public static final int ID_LOAD = 4;
+    public static final int ID_CLAIM_AND_LOAD = 4;
```

Comment:
any reason you didn't go for a new ID? have you tries trying to load a chunk already claimed? could one accidentally claim a chunk of another person this way or load it up for them?

Reply:
If you try to load an already loaded chunk nothing happens, that logic is being handled elsewhere. Same if you were to try to claim an already claimed chunk in the current version.

I can't imagine that you would be able to claim another persons chunk as that should be prevented by the same logic that prevents you claiming another person claim. But I will test that real quick to be 100% sure.

Reply:
Alright tested, you can't claim or load another persons chunks.

Reply:
Good to know. thanks